crates/bevy_render/src/renderer/graph_runner.rs

```diff
@@ -16,6 +16,17 @@ use crate::{
     renderer::{RenderContext, RenderDevice},
 };
 
+/// The [`RenderGraphRunner`] is responsible for executing a [`RenderGraph`].
+///
+/// It will run all nodes in the graph sequentially in the correct order (defined by the edges).
+/// Each [`Node`](crate::render_graph::node::Node) can run any arbitrary code, but will generally
+/// either send directly a [`CommandBuffer`] or a task that will asynchronously generate a [`CommandBuffer`]
+///
+/// After running the graph, the [`RenderGraphRunner`] will execute in parallel all the tasks to get
+/// an ordered list of [`CommandBuffer`]s to execute. These [`CommandBuffer`] will be submitted to the GPU
+/// sequentially in the order that the tasks were submitted. (which is the order of the [`RenderGraph`])
+///
+/// `CommandBuffer`: wgpu::CommandBuffer
```

Comment:
This line seems to be making Clippy angry. I think you need to wrap it in square brackets if you want to make it a link.

```suggestion
/// [`CommandBuffer`]: wgpu::CommandBuffer
```